decidim-admin/app/controllers/decidim/admin/features/base_controller.rb

```diff
@@ -8,7 +8,6 @@ module Features
       class BaseController < Admin::ApplicationController
         skip_authorize_resource
         include Concerns::ParticipatoryProcessAdmin
-        include NeedsParticipatoryProcess
```

Comment:
Controversial change, but I understand the rationale behind it. Now features don't really depend on participatory process. We'll have to be careful about specifying internal APIs that don't couple features with participatory processes though (which I think we've quite successfully done to the moment).